src/struct/commands/arguments/Argument.js

```diff
@@ -366,7 +366,8 @@ class Argument {
             return null;
         }
 
-        if (typeof type === 'function') {
```

Comment:
Fix this spacing.

Reply:
I didn't even realise I did that! Should be all good now.

Reply:
Actually, this isn't right, Don't remove the `typeof`.

Reply:
My apologies. I'm perplexed; I don't have any memory of deleting that.